robottelo/ui/base.py

```diff
@@ -539,11 +540,14 @@ def click(self, locator, wait_for_ajax=True,
         :raise: UINoSuchElementError if the element could not be found.
 
         """
-        element = self.wait_until_element(locator, timeout=waiter_timeout)
+        if type(locator).__name__ == 'tuple':
+            element = self.wait_until_element(locator, timeout=waiter_timeout)
+        else:
+            element = locator
         if element is None:
             raise UINoSuchElementError(
-                '{0}: element with locator {1} not found while trying to click'
-                .format(type(self).__name__, locator)
+                '{0}: element was not found while trying to click'
```

Comment:
Actually, we cannot as we get to that point only in case of `element==None`, so we have no access to any information here.
Of course, in case of `locator` we still can show that information, but that will require some more code.
I will check that. Thanks